test/ci/test-ov-verification.sh

```diff
@@ -0,0 +1,142 @@
+#! /bin/bash
+
+set -euo pipefail
+
+source "$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/utils.sh"
+
+# Define new owner service for testing wrong owner rejection
+new_owner_service_name=new_owner
+new_owner_dns=new_owner
+# needed for 'start_services' do not remove
+#shellcheck disable=SC2034
+new_owner_ip=127.0.0.1
+new_owner_port=8045
+new_owner_pid_file="${pid_dir}/new_owner.pid"
+new_owner_log="${logs_dir}/${new_owner_dns}.log"
+# key crt pub and subj variables are required to generate certificates
+new_owner_key="${certs_dir}/new_owner.key"
+#shellcheck disable=SC2034
+new_owner_crt="${new_owner_key/\.key/.crt}"
+new_owner_pub="${new_owner_key/\.key/.pub}"
+#shellcheck disable=SC2034
+new_owner_subj="/C=US/O=FDO/CN=New Owner"
+new_owner_service="${new_owner_dns}:${new_owner_port}"
+new_owner_url="http://${new_owner_service}"
+# needed for 'wait_for_services_ready' do not remove
+#shellcheck disable=SC2034
+new_owner_health_url="${new_owner_url}/health"
+
+start_service_new_owner () {
+  run_go_fdo_server owner ${new_owner_service} new_owner ${new_owner_pid_file} ${new_owner_log} \
+    --owner-key="${new_owner_key}" \
+    --device-ca-cert="${device_ca_crt}"
+}
```

Comment:
Source `test-resale.sh` so you don't need to add all the definitions again:
```suggestion
source "$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/test-resale.sh"
```